README.md

```diff
@@ -1,2 +1,70 @@
-# service
+# Eik HTTP Service
+
 Eik REST API - As a standalone running HTTP service
+
+## Installation
+
+```
+npm install @eik/service
+```
+
+## Usage
+
+This server can either be run as a Node executable, or as a Fastify plugin.
+
+```sh
+npx @eik/service
+```
+
+For a production setup, the Fastify plugin method is recommended.
+
+```js
+import fastify from 'fastify';
+import Service from '@eik/service';
+import Sink from '@eik/sink-gcs';
+
+// Set up the Google Cloud Storage sink
+// https://github.com/eik-lib/sink-gcs?tab=readme-ov-file#example
+const sink = new Sink({
+    credentials: {
+        client_email: 'a@email.address',
+        private_key: '[ ...snip... ]',
+        projectId: 'myProject',
+    },
+});
+
+// Set up the Eik service as a plugin
+const service = new Service({ sink });
+
+// Set up Fastify
+const app = fastify({
+    ignoreTrailingSlash: true,
+    modifyCoreObjects: false,
+    trustProxy: true,
+});
+
+// Register the Eik service in Fastify
+app.register(service.api());
+
+// Append custom HTTP ready checks
+app.get('/_/health', (request, reply) => {
+    reply.send('ok');
+});
+
+app.get('/_/ready', (request, reply) => {
+    reply.send('ok');
+});
```

Comment:
```suggestion
```
I don't feel this is important to have here, people can do what they want here and those who require ready checks would know what to do.